### PR TITLE
Update balances block

### PR DESCRIPTION
**Pull request by blackinno**
- Add configuration percent to each row: "Daily Percent".
- Rename "Balances" block to "Locked wallets"
- Move json file to config folder
Closes https://github.com/mesg-foundation/atd/issues/27

### CODE REVIEW

File: frontend/store/token.js [hunk 1]
```diff
@@ -1,38 +1,53 @@
...
+
+const abi = require('../config/abi.json')
+const abiHistoric = require('../config/abiHistoric.json')
+const contracts = require('../config/atd.json')
+const accounts = require('../config/accounts.json')
 const PRECISION = 1e18
 const web3 = new Web3(process.env.ETHEREUM_NODE)
-
-const EMPTY_TX = '0x0000000000000000000000000000000000000000000000000000000000000000'
+const ATD = '0x7be1a4d7ea5b1a9e74762f388a06912c3ea0935b'
```
Comment (NicolasMahe) [inline frontend/store/token.js]: Could you get this value directly from `config/atd.json`?
You could add a new data to `config/atd.json` to tell which one is the latest ATD smart contract.

File: frontend/store/token.js [hunk 2]
```diff
@@ -101,5 +146,50 @@ export const actions = {
       .sort((a, b) => a.date - b.date)
     events.forEach(distributed => commit('addDistributed', { distributed }))
     return events
+  },
+  FETCH_DAILYPERCENT: async ({ commit }) => {
+    commit('resetDailyPercent')
+    try {
+      const MyContract = web3.eth.Contract(abi, ATD).methods
```
Comment (NicolasMahe) [inline frontend/store/token.js]:
```suggestion
      const atd = web3.eth.Contract(abi, ATD).methods
```

File: frontend/store/token.js [hunk 3]
```diff
@@ -101,5 +146,50 @@ export const actions = {
...
+              (p, config) =>
+                p + dtvPercentage(config.ratioDTV, config.ratioDecimals),
+              0
+            )
+        }
+      })
+      getDailyPercent.forEach(dailypercent =>
+        commit('addDailyPercent', { dailypercent })
+      )
+    } catch (error) {
```
Comment (NicolasMahe) [inline frontend/store/token.js]: I think you can remove the try / catch and let the framework manage the error?

File: frontend/store/token.js (hunk 1 above)
Comment (blackinno) [reply frontend/store/token.js]: Okay i will try

File: frontend/store/token.js (hunk 3 above)
Comment (blackinno) [reply frontend/store/token.js]: Okay, I will remove try/catch but I don't sure about NuxtJS can manage the error

**Comment by blackinno** [comment]
@antho1404 @NicolasMahe  Please review again.


> I got few issues on the preview https://deploy-preview-38--mesg-atd.netlify.com/
> 
> * ReferenceError: MyContract is not defined
> * TypeError: Cannot read property 'dailyPercent' of undefined

About the issues are fixed. https://deploy-preview-38--mesg-atd.netlify.com/

**Comment by NicolasMahe** [comment]
@blackinno is it normal that the background of the graph and distribution block are gone?

**Comment by blackinno** [comment]
> @blackinno is it normal that the background of the graph and distribution block are gone?
> 
> otherwise, good job at Refactoring the getdailypercent code 👍

@NicolasMahe Yes, it's normal for now, because I will fix layout inside the component.

**Comment by blackinno** [comment]
@NicolasMahe It's fixed. Please review again. https://deploy-preview-38--mesg-atd.netlify.com/

File: frontend/store/token.js [hunk 4]
```diff
@@ -1,38 +1,40 @@
...
-    return {
-      ...account,
-      distributed: distributed,
-      balance: account.initialBalance - distributed
-    }
-  })
+  accounts: state =>
+    accounts.map(account => {
+      const distributed = state.distributed.reduce((prev, x) => prev + (x.lockedWallet === account.address ? x.amount : 0), 0)
+      const dailyPercentage = state.dailyPercentage.filter(daily => daily.account.toLowerCase() === account.address.toLowerCase())
```
Comment (NicolasMahe) [inline frontend/store/token.js]: should use `find` instead of `filter` but not blocking for this PR.

Comment (blackinno) [reply frontend/store/token.js]: Okay I will make a new PR this

**Pull request merged**